source/core/NumericString.cpp

```diff
@@ -2593,15 +2646,28 @@ void ScanSpreadsheet(StringRef inputString, StringRef formatString, StringRef de
     }
 }
 //-------------------------------------------------------------------
-VIREO_FUNCTION_SIGNATURE4(SpreadsheetStringtoArray, StringRef, StringRef, StringRef, TypedArrayCoreRef)
+VIREO_FUNCTION_SIGNATURE4(SpreadsheetStringtoArray, StringRef, StringRef, TypedArrayCoreRef, TypedArrayCoreRef)
```

Comment:
Shouldn't the fourth parameter still be a StringRef?

Reply:
Yes, I made this change before I made a separate function to handle the array case; I can back out of that.

Reply:
Fixed.